cache/FIFO_FH.h

```diff
@@ -606,12 +620,13 @@ bool FIFO_FHCache<TKey, TValue, THash>::find(TValue& ac,
     // we can re-check this and remove it if possible
 
     // Generally when we found the thing in frozen cache
-    if(m_fasthash->find(key, ac) && (ac != nullptr))
+    std::unique_lock<ListMutex> lockMap(m_mapMutex);
```

Comment:
Similar comment above, I just want to check do we need this lock, since at this point chunkMapper is never being written to? And then if so, I guess using CLHT would be better.